src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/c1/Runtime1.java

```diff
@@ -62,6 +62,6 @@ public Address entryFor(int id) {
   /** FIXME: consider making argument "type-safe" in Java port */
   public CodeBlob blobFor(int id) {
     Address blobAddr = blobsField.getStaticFieldAddress().getAddressAt(id * VM.getVM().getAddressSize());
-    return VM.getVM().getCodeCache().createCodeBlobWrapper(blobAddr);
```

Comment:
We don't need to change all the callers if we keep a 1-arg version of createCodeBlobWrapper():
```
public CodeBlob createCodeBlobWrapper(Address codeBlobAddr) {
    return createCodeBlobWrapper(codeBlobAddr, codeBlobAddr);
}
```